documents/GIT_POLICY.md

```diff
@@ -13,10 +13,36 @@
 
 **The number of branches depend on the specific project*
 
-### Git flow
-- Start a feature from code base branch: `[dev] $ git checkout -b feat/new-feature`
-- Create a pull request to finish a feature
-- TL/PM/Senior Dev will check such PRs and then merge to dev branch
+### **Protected branches:** 
+We should have protected branches.
+  Which is always not easy to be merged. They can be different name in different projects.
+  For example, in this document, we have branches: `master` and `product`.
+  `master` should include well tested code,
+  feature branches need to be reviewed by other devs before merged into `master`.
+  And `product` should be the final result after each milestone we reach
+  and be ready for production.
+### **Feature branches:** 
+the branches the developers be working on it.
+  Include bug fix, new feature.
+  Every feature branch should be corresponding with an issue be created on git server
+  (github ,gitlab , etc...). A `feature branch` should be named as a standard format.
+  I would love to propose a standard: `{issue_id}_{issue_name}` in this document.
+  For example: `1_init_source_base`, `205_fix_bug_when_send_marketing_email`, ...
+### **Issue:** 
+an issue should be full detail and informative. If it is a bug.
+  It should give the link to the bug, describe how to reproduce the bug,
+  the old state and the expected state.
```

Comment:
Issue không phổ biến trong các dự án của team hiện tại, các task/bug thường được quản lý thông qua jira. Có thể thêm mô tả cho phần phù hợp với các dự án quản lý hoàn toàn thông qua github